cookbooks/system/ruby/recipes/install.rb

```diff
@@ -1,22 +1,30 @@
-parcel 'ruby2.0'
-parcel 'ruby2.0-dev'
+execute "add-apt-repository ppa:brightbox/ruby-ng" do
```

Comment:
[Yes](stackoverflow.com/questions/9576124). I believe we already have [apt here](https://github.com/funkensturm/warehouse/tree/master/cookbooks/providers/apt).